automatic/autoit.install/update.ps1

```diff
@@ -0,0 +1,46 @@
+import-module au
+
+$releases = "https://www.autoitscript.com/site/autoit/downloads"
+
+if ($MyInvocation.InvocationName -ne '.') { # run the update only if script is not sourced
+    function global:au_BeforeUpdate {
+        Remove-Item "$PSScriptRoot\tools\*.exe"
+
+        $filePath = "$PSScriptRoot\tools\$($Latest.FileName)"
+        Invoke-WebRequest $Latest.URL32 -OutFile $filePath
+
+        $Latest.ChecksumType = "sha256"
+        $Latest.Checksum = Get-FileHash -Algorithm $Latest.ChecksumType -Path $filePath | ForEach-Object Hash
+    }
+}
+
+function global:au_SearchReplace {
+    @{
+        "tools\chocolateyInstall.ps1" = @{
+            "(^[$]filePath\s*=\s*`"[$]toolsDir\\)(.*)`"" = "`$1$($Latest.FileName)`""
+        }
+        ".\tools\verification.txt" = @{
+            "(?i)(1\..+)\<.*\>" = "`${1}<$($Latest.URL32)>"
+            "(?i)(checksum type:\s+).*" = "`${1}$($Latest.ChecksumType)"
+            "(?i)(checksum:\s+).*" = "`${1}$($Latest.Checksum)"
+        }        
+     }
+}
+
+function global:au_GetLatest {
```

Comment:
I would move this function to the `autoit\update.ps1` file and aquire both the install url and portable url at the same time have have this package and the portable package call the meta package update file instead (getting rid of duplicated logic).